aqua/input-leap/Portfile

```
@@ -0,0 +1,78 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        input-leap input-leap 198779cf9e798aac6ad41427e32d031305736de3
+version             20230416
+revision            0
+categories          aqua net sysutils
+platforms           darwin
+license             GPL-2
```

From the project's [LICENSE](https://github.com/input-leap/input-leap/blob/198779cf9e798aac6ad41427e32d031305736de3/LICENSE#L7-L8) file:

> This program is released under the GPL with the additional exemption
> that compiling, linking, and/or using OpenSSL is allowed.

Therefore:

```suggestion
license             {GPL-2 OpenSSLException}
```